Docker/docker-compose.yml

```diff
@@ -91,6 +93,14 @@ services:
       options:
         max-size: "200k"
         max-file: "10"
+  adaguc-redis:
+    image: redis:alpine
+    container_name: adaguc-redis
+    hostname: adaguc-redis
+    networks:
+      - adaguc-network
+    expose:
+      - "${REDIS_PORT}"
```

Comment:
REDIS_PORT should be written to the .env file in the `Docker/docker-compose-generate-env.sh` script

Reply:
Removed